examples/financialcircuit/README.md

```diff
@@ -88,7 +88,7 @@ Counter parties don't know each other quotes.
 
 ## Test
 Verifier smart contract was deployed on Rinkeby for tests purpose. Here is the Etherscan link:
-https://rinkeby.etherscan.io/address/0xad6ba91bd376050e807b67cd2ff83a638b3095c2#code
+https://mumbai.polygonscan.com/address/0x14e1d3c53a42b64dde5e78b069bbd295270117b9#code
 
 You can use remix to test it since Etherscan form to Write parameters didin't work with 2 dymension array.
```

Comment:
Is this comment still valid? If yes, please fix typos in didn't and dymension. If no, please remove.

Reply:
Comment is still valid. It isn't possible to use READ function from Polygonscan. I tested it using Remix.

Reply:
Typos were fixed.